From 7c5b3bfbd10bd083a0813b841b8ecd821f50f748 Mon Sep 17 00:00:00 2001
From: Dimitrios Katsaros <patcherwork@gmail.com>
Date: Fri, 17 Jan 2020 14:16:23 +0100
Subject: [PATCH] decodebin3: Reduced logging level of messages

The logging is set to warning for a drain event, which is part of the
normal functionality of the parsebin.
---
 gst/playback/gstdecodebin3.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gst/playback/gstdecodebin3.c b/gst/playback/gstdecodebin3.c
index cfe061a..02809ad 100644
--- a/gst/playback/gstdecodebin3.c
+++ b/gst/playback/gstdecodebin3.c
@@ -753,7 +753,7 @@ parsebin_drained_cb (GstElement * parsebin, DecodebinInput * input)
   gboolean all_drained;
   GList *tmp;
 
-  GST_WARNING_OBJECT (dbin, "input %p drained", input);
+  GST_INFO_OBJECT (dbin, "input %p drained", input);
   input->drained = TRUE;
 
   all_drained = dbin->main_input->drained;
@@ -764,7 +764,7 @@ parsebin_drained_cb (GstElement * parsebin, DecodebinInput * input)
   }
 
   if (all_drained) {
-    GST_WARNING_OBJECT (dbin, "All inputs drained. Posting about-to-finish");
+    GST_INFO_OBJECT (dbin, "All inputs drained. Posting about-to-finish");
     g_signal_emit (dbin, gst_decodebin3_signals[SIGNAL_ABOUT_TO_FINISH], 0,
         NULL);
   }
